test/spec/features/snapping/BpmnConnectSnappingSpec.js

```diff
@@ -394,4 +394,40 @@ describe('features/snapping - BpmnConnectSnapping', function() {
 
   });
 
+
+  describe('association', function() {
+
+    describe('sequence flow', function() {
+
+      it('should snap target', inject(function(connect, dragging, elementRegistry) {
+
+        // given
+        var TextAnnotation = elementRegistry.get('TextAnnotation_1'),
+            sequenceFlow = elementRegistry.get('Flow_1'),
+            sequenceFlowGfx = elementRegistry.getGraphics(sequenceFlow);
+
+        // when
+        connect.start(canvasEvent({ x: 470, y: 1340 }), TextAnnotation);
+
+        dragging.hover({ element: sequenceFlow, gfx: sequenceFlowGfx });
+
+        dragging.move(canvasEvent({ x: 400, y: 1455 }));
+
+        dragging.end();
+
+        // then
+        var waypoints = TextAnnotation.outgoing[0].waypoints;

```

Comment:
Rename to `textAnnotation`.

Reply:
done in [8a5578b](https://github.com/bpmn-io/bpmn-js/pull/1661/commits/8a5578b70d2aab539f8759313140fc53c2435a43)